core/commands/custom.py

```diff
@@ -27,8 +27,17 @@ class GsCustomCommand(WindowCommand, GitCommand):
     """
 
     def run(self, **kwargs):
+        if '{PROMPT_ARG}' in kwargs.get('args'):
+            self.last_kwargs = kwargs
+            msg = kwargs.pop("prompt_msg", "Command argument: ")
+            self.window.show_input_panel(msg, "", self.custom_argument_entered,
```

Comment:
It seems that this could be simplified into a single `lambda`, eliminating the `custom_argument_entered` method below.